tools/AutoTuner/README.md

```diff
@@ -0,0 +1,205 @@
+[here](https://cloud.google.com/kubernetes-engine/docs/quickstart). The
+quickstart guide will show how to enable GKE (Google's Kubernetes Engine)
+start a CLI interface and get the settings for your project.
+
+## Create a Kubernetes cluster
+
+Create GKE cluster using the following `gcloud` command as a guide.
+Note that each argument defines a characteristic of the cluster, further
+the cluster and node pool will allocate resources that cost money, so be
+mindful when choosing the configuration. For additional information see:
+
+```bash
+gcloud beta container clusters create --help
+gcloud beta container node-pools create --help
+```
+
+```bash
+gcloud beta container clusters create "autotuner" \
+         --machine-type "e2-standard-8" \
+         --image-type "UBUNTU_CONTAINERD" \
+         --disk-type "pd-standard" \
+         --disk-size "100" \
+         --num-nodes "1" \
+         --enable-autoscaling \
+         --min-nodes "1" \
+         --max-nodes "10"
+```
+
+Configure `gcloud` to use the Kubernetes credentials of the newly created
+cluster. The credentials allow the use of `kubectl` locally.
+
+```bash
+gcloud components install kubectl
+gcloud container clusters get-credentials autotuner
+```
+
+Create a new node pool. This step is optional, but recommended. The new pool
+will use preemptive nodes which are cheaper, but also has a more powerful
+CPU and have more disk space.
+
+```bash
+gcloud beta container node-pools create "worker-pool" \
+         --cluster "autotuner" \
+         --machine-type "c2-standard-60" \
+         --image-type "UBUNTU_CONTAINERD" \
+         --disk-type "pd-standard" \
+         --disk-size "2000" \
+         --preemptible \
+         --num-nodes "3" \
+         --enable-autoscaling \
+         --min-nodes "1" \
+         --max-nodes "625"
+```
+
+## Create NFS and setup mount options
+
+NOTE: This tutorial requires a working NFS server.
+
+To access the NFS mount point in the Ray cluster we use a NFS helm chart. You
+need to modify the helm chart with the information about your server IP
+and mount point path.
+
+### Install NFS Kubernetes provisioner
+
+```bash
+helm repo add nfs-subdir-external-provisioner https://kubernetes-sigs.github.io/nfs-subdir-external-provisioner
+helm repo update
+```
+
+Before installing the helm chart, double check the fields marked with
+`TODO` on the file `./nfs/nfs-helm-values.yaml`. The `server` and `path`
+need to match the NFS server IP and the path to the exposed folder.
+The `nodeSelector` must match the non-preemptive Kubernetes pool.
+
+```bash
+helm install nfs nfs-subdir-external-provisioner/nfs-subdir-external-provisioner -f nfs/nfs-helm-values.yaml
+```
+
+### Create a PVC
+
+Create the PVC:
+
+```bash
+kubectl create -f nfs/nfs-pvc.yaml
+```
+
+## Ray cluster
+
+The two main files for the helm chart are:
+
+```bash
+helm-chart/templates/raycluster.yaml
+helm-chart/values.yaml
+```
+
+To restrict to a specific node pool, replace POOL_NAME with the name
```

Comment:
restrict what? do you mean "allocate" or "configure" etc.

Reply:
Fixed